index.html

```diff
@@ -586,14 +601,24 @@
     var scheduleElement = document.getElementById('mainschedule');
     scheduleElement.addEventListener("swiped", scheduleSwiped, false);
     updateMainSchedule();
+    document.querySelector("paper-typeahead-input").addEventListener('pt-item-confirmed', function() {
+      var obj = document.querySelector("paper-typeahead-input").inputObject;
+      var name = obj.name.replace(" ", "_");
+      if (obj.prefix == "student") {
+        requestAdditionalData(name, "student", renderStudent);
+      } else {
+        requestAdditionalData(name, "teacher", renderTeacher);
+      }
+      openPopup(obj.name);
+    });
   });
 
   if (showPrivacyDialog) {
     setTimeout(function(){ document.getElementById("privacydialog").open(); }, 0);
   }
   </script>
 
-  <style>
+  <style is="custom-style">
```

Comment:
It allows me to set the padding of `paper-toolbar ::shadow #topBar` to 0px without having to modify the element.